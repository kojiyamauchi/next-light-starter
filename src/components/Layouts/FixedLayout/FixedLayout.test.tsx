/*
  Fixed Layout Component Unit Test.
*/

import { FixedLayout } from './'
import { render /* ,screen */ } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
// import userEvent from '@testing-library/user-event'

describe('Fixed Layout Component Unit Test', () => {
  it('Snap Shot Testing', () => {
    const { asFragment } = render(<FixedLayout pages="ui-test" />)
    expect(asFragment()).toMatchSnapshot()
  })
})
