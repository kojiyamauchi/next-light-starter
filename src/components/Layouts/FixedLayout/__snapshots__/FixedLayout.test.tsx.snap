// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fixed Layout Component Unit Test Snap Shot Testing 1`] = `
<DocumentFragment>
  <div
    class="ui-test wrapper"
  >
    <header
      class="wrapper"
    >
      <h1
        class="primary-heading"
      >
        Next.js light Starter
      </h1>
      <ul
        class="wrapper"
      >
        <li>
          <a
            class="link"
            href="/"
          >
            Primary Page
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/secondary"
          >
            Secondary Page
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/third"
          >
            Third Page
          </a>
        </li>
        <li>
          <a
            class="link"
            href="/blog-list"
          >
            Blog Page
          </a>
        </li>
      </ul>
    </header>
    <main
      class="main-content"
      style="background-image: url(/next-light-starter/webps/logo_cream.webp);"
    />
    <footer
      class="wrapper"
    >
      <a
        class="link"
        href="http://kojiyamauchi.com"
        rel="noreferrer"
        target="_blank"
      >
        Â© 2021 All The Small Things, All Rights Reserved.
      </a>
    </footer>
  </div>
</DocumentFragment>
`;
